.eslintrc

```
@@ -0,0 +1,18 @@
+{
+    "root": true,
+    "extends": ["plugin:@18f/eslint-plugin-identity/recommended"],
+    "plugins": ["@18f/eslint-plugin-identity"],
+    "env": {
+      "browser": true,
+      "jest": true
+    },
+    "parserOptions": {
+      "ecmaVersion": 2021,
+      "sourceType": "module"
+    },
+    "rules": {
+      // ESLint is currently unable to resolve packages with the exports directive
+      // See: https://github.com/import-js/eslint-plugin-import/issues/1810
+      "import/no-unresolved": ["error", { "ignore": ["@18f/identity-address-search", "@18f/identity-i18n", "@18f/identity-components"] }]
+    }
```

Can we see if it works without these overrides? I think they were for some specific usage in `identity-site`.

```suggestion
    }
```